.vscode/settings.json

```diff
@@ -4,14 +4,14 @@
     "prettier.requireConfig": true,
     "editor.rulers": [80],
     "editor.codeActionsOnSave": {
-        "source.fixAll": false,
-        "source.organizeImports": true
-      },
-      "[typescript]": {
+        "source.fixAll": "never",
+        "source.organizeImports": "explicit"
+    },
+    "[typescript]": {
         "editor.defaultFormatter": "esbenp.prettier-vscode"
-      },
```

Comment:
I didn't intend to make changes to this file. I should revert this.

Reply:
it appears unchanged

Reply:
I put it back to the original